vertex_launcher/src-tauri/src/env.rs

```diff
@@ -111,7 +111,7 @@ pub(crate) const LOG_TIMEZONE: tauri_plugin_log::TimezoneStrategy =
 
 /// The UPDATE_RATE is the period used to advertise to the frontend
 /// the download progression.
-pub(crate) const UPDATE_RATE: u64 = 1000; // 1000ms
+pub(crate) const UPDATE_RATE: u64 = 100; // default: 1000ms
```

Comment:
Change comment 

```suggestion
pub(crate) const UPDATE_RATE: u64 = 100; // default: 100ms
```

Maybe downgrade to u8 or u16 for better ram management?

Reply:
resolved by https://github.com/GuicLuca/VertexGameLauncher/pull/5/commits/466f9f05f8f09cadae8dcaf59e3d03a87f7fb31a